chart/templates/statefulset.yaml

```diff
@@ -118,11 +117,20 @@ spec:
             {{- end }}
             {{- end }}
             {{- end }}
+            {{- end }}
+
+            {{- if .Values.standalone.tls }}
+            {{- range $secretName := .Values.standalone.tls }}
```

Comment:
Once the `tls` is `true`, you should be able to set ENV below:

- "BYDB_TLS", which enables the server's TLS function.
- "BYDB_CERT_FILE", which is the path of cert file 
- "BYDB_KEY_FILE", which is the path of private key file

Reply:
`.Values.standalone.tls` should not be an array. It has got to be an individual value.